config/environment.rb

```diff
@@ -1,12 +1,31 @@
-ENV['SINATRA_ENV'] ||= "development"
+ENV['SINATRA_ENV'] ||= 'development'
+ENV['SESSION_SECRET'] = 'Oq<dDrr=C1~pI*]MK:q:'
 
 require 'bundler/setup'
 require 'rack-flash'
-Bundler.require(:default, ENV['SINATRA_ENV'])
+Bundler.require()
 
-ActiveRecord::Base.establish_connection(
-  :adapter => "sqlite3",
-  :database => "db/#{ENV['SINATRA_ENV']}.sqlite"
-)
+configure :development do
+  # set :database, 'sqlite:///dev.db'
+  # set :show_exceptions, true
+  ActiveRecord::Base.establish_connection(
+    :adapter => "sqlite3",
+    :show_exceptions => true,
+    :database => "db/#{ENV['SINATRA_ENV']}.sqlite"
+  )
+end
+ 
+configure :production do
+  db = URI.parse(ENV['DATABASE_URL'] || 'postgres:///localhost/mydb')
```

Comment:
does this work with `postgres://localhost`?

Reply:
ve not created the db url yet. since its production. only test which is sqllite works

Reply:
You have a development configuration